packages/app-interbit-io/src/containers/Developers/Resources.js

```diff
@@ -35,7 +35,7 @@ class DevelopersResources extends Component {
                 )}
                 {s.callToActions.map(a => (
                   <p key={a.to}>
-                    <a href={a.to} target="_blank">
+                    <a href={a.to} target="_blank" rel="noopener noreferrer">
                       {a.to}
```

Comment:
I had to look this up to see what it did. Should be included in the PR comment

Reply:
👍